client_app/src/components/volunteer/Shelters.js

```diff
@@ -12,6 +12,19 @@ import { useSpring, animated } from "@react-spring/web";
 import dayjs from 'dayjs';
 import CurrentSelection from "./CurrentSelection";
 
+// hardcoded predefined shifts
+const PREDEFINED_SHIFTS = {
+  "shelter-1": [
+    { id: 1, start: "2025-02-17T09:00:00", end: "2025-02-17T13:00:00", title: "Morning Shift" },
+    { id: 2, start: "2025-02-17T13:00:00", end: "2025-02-17T17:00:00", title: "Afternoon Shift" },
+    { id: 3, start: "2025-02-17T17:00:00", end: "2025-02-17T21:00:00", title: "Evening Shift" }
+  ],
+  "shelter-2": [
+    { id: 4, start: "2025-02-17T08:00:00", end: "2025-02-17T12:00:00", title: "Early Shift" },
+    { id: 5, start: "2025-02-17T12:00:00", end: "2025-02-17T16:00:00", title: "Mid-day Shift" }
```

Comment:
The timestamps should be in milliseconds since the Epoc in UTC (that's what the server will be sending you eventually)

Reply:
I recommend placing the hard-coded data into its own file, and import it in the IndividualShelter.js